src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_InvalidExpression.cs

```diff
@@ -33,9 +33,7 @@ static void Main(string[] args)
             string expectedOperationTree = @"
 IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: 'Console.WriteLine2()')
   Children(1):
-      IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: 'Console.WriteLine2')
```

Comment:
>IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: 'Console.WriteLine2') [](http://example.com/codeflow?start=6&length=92)

This change feels unexpected.

Reply:
This is due to the change in `BindMemberAccessWithBoundLeft`, which no longer returns `null` from `tryBindMemberAccessWithBoundTypeLeft` in most error cases. As a result, for those scenarios, it no longer calls `BindMemberAccessBadResult` (which wrapped the `BoundTypeExpression` into a `BoundBadExpression`).
The resulting bound tree (and IOperation tree) seems improved as a result. We know what the receiver, `Console`, means and there is nothing wrong with that receiver.

New:
```
call
├─receiverOpt
│ └─typeExpression
│   ├─aliasOpt
│   ├─boundContainingTypeOpt
│   ├─boundDimensionsOpt
│   ├─typeWithAnnotations: System.Console
│   └─type: System.Console
├─method: System.Console.WriteLine2()
├─arguments
├─argumentNamesOpt: (null)
├─argumentRefKindsOpt: (null)
├─isDelegateCall: False
├─expanded: False
├─invokedAsExtensionMethod: False
├─argsToParamsOpt: (null)
├─defaultArguments: Microsoft.CodeAnalysis.BitVector
├─resultKind: Empty
├─originalMethodsOpt: {}
├─type: ?
└─hasErrors: True
```

Old:
```
call
├─receiverOpt
│ └─badExpression
│   ├─resultKind: Empty
│   ├─symbols: {}
│   ├─childBoundNodes
│   │ └─typeExpression
│   │   ├─aliasOpt
│   │   ├─boundContainingTypeOpt
│   │   ├─boundDimensionsOpt
│   │   ├─typeWithAnnotations: System.Console
│   │   └─type: System.Console
│   ├─type: ?
│   └─hasErrors: True
├─method: ?.()
├─arguments
├─argumentNamesOpt: (null)
├─argumentRefKindsOpt: (null)
├─isDelegateCall: False
├─expanded: False
├─invokedAsExtensionMethod: False
├─argsToParamsOpt: (null)
├─defaultArguments: Microsoft.CodeAnalysis.BitVector
├─resultKind: NotInvocable
├─originalMethodsOpt: {}
├─type: ?
└─hasErrors: True
```